### DISCUSSION TITLE
Can you add the version to the appimage file?

### DISCUSSION THREAD

**Discussion by catpreneur** [mifi/lossless-cut#924] [Ideas]
Currently all appimage releases have the filename `LosslessCut-linux.AppImage` but it makes it impossible to tell which version i got when i have like 3 versions downloaded from the past. The common practice is to put the version number in the filename.

**Comment by mifi** [comment]
The problem is that if I include the version inside the name, I can no longer link to the newest version from the github front page https://github.com/mifi/lossless-cut/releases/latest/download/LosslessCut-linux.AppImage

I'm not sure how to fix that if there's a version inside the name

**Comment by catpreneur** [comment]
You can actually just link to `https://github.com/username/repo/releases/latest` and it'll redirect to the latest

**Comment by catpreneur** [comment]
I just realized what you meant. You wanted to link the actual file from the frontpage. In that case, I would advise against that.
Just link to the release page and let people pick what to download. People will want to see what version it is they're downloading. This is normal btw.

**Comment by mifi** [comment]
I had people in the past who were confused by all the options on the download page itself, so I think being able to describe the download options more concisely on the front page has a value
